cypress/support/stackable/helpers/index.js

```diff
@@ -411,3 +411,56 @@ export const assertContainer = ( selector, options = {}, attrNameTemplate = 'col
 			.assertClassName( selector, 'ugb--shadow-9' )
 	} )
 }
+
+/*
+* Helper function for Container link panel assertion.
+*
+* @param {string} selector
+* @param {Object} options
+* @param {Object} assertOptions
+*/
+export const assertContainerLink = ( selector, options, assertOptions = {} ) => {
+	const {
+		viewport,
+	} = options
+	const desktopOnly = callback => viewport === 'Desktop' && callback()
+
+	const _collapse = ( collapseName = '' ) => {
```

Comment:
`_collapse` and `_adjust` functions are no longer needed in this case. Take note that we only do "conditional" adjustments to avoid failing the tests if the option is not present inside a specific block's inspector. Just make sure that we only call `assertContainerLink` to blocks that support that module.

Reply:
i added these conditional adjustments because if there is more than 1 column, the labels of the controls will be:
Link / URL #1, Link / URL #2
etc. what I did is to conditionally set the columns to 1 so that container link will be tested for a singular column only (e.g. Link / URL)

Reply:
```
cy.getBlockAttributes().then( ( { columns } ) => {
    if ( columns && columns !== 1 ) {
         // adjust with Link / URL # 1
         return
    }
    // adjust with Link / URL
} )
```

Reply:
> i added these conditional adjustments because if there is more than 1 column, the labels of the controls will be:
> Link / URL # 1, Link / URL # 2
> etc. what I did is to conditionally set the columns to 1 so that container link will be tested for a singular column only (e.g. Link / URL)

Did you use `Link / URL` in your commit? I can't see it